src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -621,9 +621,10 @@ private function buildTableRows(array $rows): TableRows
                 if (!strstr($cell ?? '', "\n")) {
                     continue;
                 }
-                $escaped = implode("\n", array_map([OutputFormatter::class, 'escapeTrailingBackslash'], explode("\n", $cell)));
+                $eol = strstr($cell ?? '', "\r\n") ? "\r\n" : "\n";
```

Comment:
AppVevor seems to work on a Windows OS but the content were using `\n`, so I think it's better to do this way:
- If the content of the cell is using `\r\n`, we're using it.
- If the content of the cell is using `\n` we're keeping the old behavior.

The behavior doesn't depend on the OS but on the content of the cell. Since the purpose is just to add extra spaces, we're keeping the same EOL character than the one used.

Reply:
let's use strpos instead of strstr

Reply:
Done, I changed only the code I added to avoid conflict.

`fabbot.io` is asking to change this to `str_contains` but I'm not sure I can since Symfony 5.4 allows PHP 7.2

Reply:
ah good call, str_contains is used already in the component so let's use it